MapboxIoTMap/app/build.gradle

```diff
@@ -1,12 +1,11 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.0"
+    compileSdkVersion 26
     defaultConfig {
         applicationId "com.mapbox.iotmap"
-        minSdkVersion 24
-        targetSdkVersion 25
+        minSdkVersion 27
```

Comment:
Why is minSdk higher than targetSdk?

Reply:
@electrostat good catch - I have no idea why the default set up did that. I just fixed it with eb9f3348cef4450fcfda7cce7e2bc696a5608c58.